pkg/config/otlp.go

```diff
@@ -5,36 +5,85 @@
 
 package config
 
-// Experimental OTLP configuration paths.
+import (
+	"net"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// OTLP configuration paths.
 const (
-	ExperimentalOTLPSection           = "experimental.otlp"
-	ExperimentalOTLPHTTPPort          = ExperimentalOTLPSection + ".http_port"
-	ExperimentalOTLPgRPCPort          = ExperimentalOTLPSection + ".grpc_port"
-	ExperimentalOTLPTracePort         = ExperimentalOTLPSection + ".internal_traces_port"
-	ExperimentalOTLPMetricsEnabled    = ExperimentalOTLPSection + ".metrics_enabled"
-	ExperimentalOTLPTracesEnabled     = ExperimentalOTLPSection + ".traces_enabled"
-	ReceiverSubSectionKey             = "receiver"
-	ExperimentalOTLPReceiverSection   = ExperimentalOTLPSection + "." + ReceiverSubSectionKey
-	ExperimentalOTLPMetrics           = ExperimentalOTLPSection + ".metrics"
-	ExperimentalOTLPTagCardinalityKey = ExperimentalOTLPMetrics + ".tag_cardinality"
+	OTLPSection               = "otlp_config"
+	OTLPTracesSubSectionKey   = "traces"
+	OTLPTracePort             = OTLPSection + "." + OTLPTracesSubSectionKey + ".internal_port"
+	OTLPTracesEnabled         = OTLPSection + "." + OTLPTracesSubSectionKey + ".enabled"
+	OTLPReceiverSubSectionKey = "receiver"
+	OTLPReceiverSection       = OTLPSection + "." + OTLPReceiverSubSectionKey
+	OTLPMetricsSubSectionKey  = "metrics"
+	OTLPMetrics               = OTLPSection + "." + OTLPMetricsSubSectionKey
+	OTLPMetricsEnabled        = OTLPSection + "." + OTLPMetricsSubSectionKey + ".enabled"
+	OTLPTagCardinalityKey     = OTLPMetrics + ".tag_cardinality"
 )
 
 // SetupOTLP related configuration.
 func SetupOTLP(config Config) {
-	config.BindEnvAndSetDefault(ExperimentalOTLPTracePort, 5003)
-	config.BindEnvAndSetDefault(ExperimentalOTLPMetricsEnabled, true)
-	config.BindEnvAndSetDefault(ExperimentalOTLPTracesEnabled, true)
-	config.BindEnv(ExperimentalOTLPHTTPPort, "DD_OTLP_HTTP_PORT")
-	config.BindEnv(ExperimentalOTLPgRPCPort, "DD_OTLP_GRPC_PORT")
+	config.BindEnvAndSetDefault(OTLPTracePort, 5003)
+	config.BindEnvAndSetDefault(OTLPMetricsEnabled, true)
+	config.BindEnvAndSetDefault(OTLPTracesEnabled, true)
```

Comment:
Would it make sense to add these to the config_template, and mention that they are settable via env vars (`@env`)?

Reply:
@mx-psi will work on env. vars. exclusively in a separate PR.